docs/articles/documentation/guides/continuous-integration/github-actions-and-browserstack.md

```diff
@@ -49,6 +51,28 @@ jobs:
 
 This job runs on a GitHub-hosted virtual machine with the latest Windows version. `test` is the [job ID](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_id) that must be unique to the `jobs` object.
 
+> You can use a GitHub-hosted virtual machine with *any base OS* to run tests. For the sake of simplicity, all examples listed here run on `windows-latest`.
```

Comment:
This phrase is fine, but I guess if we change `For the sake of simplicity` -> `For simplicity` we'll sound more technical.